config/crd/bases/api-gateway.consul.hashicorp.com_gatewayclassconfigs.yaml

```diff
@@ -36,6 +36,17 @@ spec:
           spec:
             description: Spec defines the desired state of GatewayClassConfig.
             properties:
+              connectionManagement:
+                description: Configuration information for managing connections in
+                  Envoy
+                properties:
+                  maxConnections:
```

Comment:
`connectionManagement.maxConnections` is somewhat long; however, I opted **not** to abbreviate to something like `connMgmt.maxConns` since we haven't abbreviated any other parts of the spec.